integrations/huggingface-transformers/tutorials/sparse-transfer-learning-bert.md

```diff
@@ -0,0 +1,343 @@
+
+## Other Dataset Examples
+
+Let's walk through commands for other use cases. Here is an overview of some datasets we have transfered to:
+
+| Use Case                   | Dataset                                                                       | Description                                                                                                                                                                                                          | Sparse Transfer Results  |
+|----------------------------|-------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------|
+| Question Answering          | [SQuAD](https://huggingface.co/datasets/squad)                     | A reading comprehension dataset, consisting of questions posed by crowdworkers on a set of Wikipedia articles, where the answer to every question is a segment of text from the corresponding reading passage.  | 88.0 F1 (85.55 baseline)  |
+| Binary Classification       | [QQP](https://huggingface.co/datasets/glue/viewer/qqp/train)       | A dataset made up of potential question pairs from Quora with a boolean label representing whether or not the questions are duplicates. | 91.08 acc (91.06 baseline)   |
+| Multi-Class Classification  | [MultiNLI](https://huggingface.co/datasets/glue/viewer/mnli/train) | A crowd-sourced collection of sentence pairs annotated with textual entailment information. It covers a range of genres of spoken and written text and supports a distinctive cross-genre generalization evaluation. | 82.56 acc (84.53 baseline) |
+| Multi-Label Classification  | [GoEmotions](https://huggingface.co/datasets/go_emotions)          | A dataset of Reddit comments labeled for 27 emotion categories or Neural (some comments have multiple).   | 48.82 avgF1 (49.85 baseline) |
+| Sentiment Analysis          | [SST2](https://huggingface.co/datasets/conll2003)                  | A corpus that includes fine-grained sentiment labels for phrases within sentences and presents new challenges for sentiment compositionality.    | 91.97 acc (92.89 baseline) |
+| Document Classification     | [IMDB](https://huggingface.co/datasets/imdb)                       | A large movie review dataset for binary sentiment analysis. Input sequences are long (vs SST2) | 93.16 acc (94.19 baseline) |
+| Token Classification (NER)  | [CoNNL-2003](https://www.clips.uantwerpen.be/conll2003/ner/)       | A dataset concentrated on four types of named entities: persons, locations, organizations, and names of miscellaneous entities that do not belong to the previous three groups. | 98.55 acc (98.98 baseline) |
+
+## Transfer Learning the Model
+
+The following commands were used to generate the models:
```

Comment:
I think it might not be clear to a first-time user why some commands have `dataset_name` / `task_name` and others don't